### PR TITLE
[Merged by Bors] - feat: port LinearAlgebra.PiTensorProduct

### PR DESCRIPTION
**Pull request by int-y1**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by int-y1** [comment]
25 errors left

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by jjaassoonn** [comment]
bors cancel

File: Mathlib/LinearAlgebra/PiTensorProduct.lean [hunk 1]
```diff
@@ -314,7 +314,7 @@ variable {R}
 
 -- mathport name: «expr⨂ₜ[ ] , »
 /-- pure tensor in tensor product over some index type -/
-notation3:100"⨂ₜ["R"] "(...)", "r:(scoped f => tprod R f) => r
+notation "⨂ₜ[" R "]" x => tprod R x
```
Comment (jjaassoonn) [inline Mathlib/LinearAlgebra/PiTensorProduct.lean]:
```lean
notation3:100"⨂ₜ["R"] "(...)", "r:(scoped f => tprod R f) => r
```
cause a build error, not sure if this fix the error. Perhaps someone who knows about notation can fix this

**Pull request closed without merging**